src/main/java/org/folio/consortia/controller/TenantController.java

```diff
@@ -1,23 +1,31 @@
 package org.folio.consortia.controller;
 
 import lombok.RequiredArgsConstructor;
+import org.folio.consortia.domain.dto.Tenant;
+import org.folio.consortia.rest.resource.ConsortiumIdApi;
 import org.folio.consortia.service.TenantService;
 import org.folio.consortia.domain.dto.TenantCollection;
-import org.folio.consortia.rest.resource.TenantsApi;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.UUID;
+
 @RestController
 @RequestMapping("/consortia")
 @RequiredArgsConstructor
-public class TenantController implements TenantsApi {
+public class TenantController implements ConsortiumIdApi {
```

Comment:
Is it possible to have ConsortiumApi name here?

Reply:
I dont think we can customize name , 
we can make /consortia/{consortiumId} as base url . in this case we will have a meaningful api interfaces (TenantsAPi , UserTenatsApi ..etc
)

Reply:
we will fix it in scope of this ticket [MODCON-7](https://issues.folio.org/browse/MODCON-7)